sdk/healthinsights/health-insights-radiologyinsights-rest/review/health-insights-radiologyinsights.api.md

```diff
@@ -0,0 +1,1090 @@
+    location?: Array<CodeableConcept>;
+    note?: Array<Annotation>;
+    objective?: {
+        name: string;
+        type?: CodeableConcept;
+    }[];
+    partOf?: Array<Reference>;
+    period?: Period;
+    phase?: CodeableConcept;
+    primaryPurposeType?: CodeableConcept;
+    principalInvestigator?: Reference;
+    protocol?: Array<Reference>;
+    reasonStopped?: CodeableConcept;
+    resourceType: "ResearchStudy";
+    site?: Array<Reference>;
+    sponsor?: Reference;
+    status: string;
+    title?: string;
+}
+
+// @public
+export interface ResearchStudyOutput extends DomainResourceOutputParent {
+    arm?: {
+        name: string;
+        type?: CodeableConceptOutput;
+        description?: string;
+    }[];
+    category?: Array<CodeableConceptOutput>;
+    condition?: Array<CodeableConceptOutput>;
+    contact?: Array<ContactDetailOutput>;
+    description?: string;
+    enrollment?: Array<ReferenceOutput>;
+    focus?: Array<CodeableConceptOutput>;
+    identifier?: Array<IdentifierOutput>;
+    keyword?: Array<CodeableConceptOutput>;
+    location?: Array<CodeableConceptOutput>;
+    note?: Array<AnnotationOutput>;
+    objective?: {
+        name: string;
+        type?: CodeableConceptOutput;
+    }[];
+    partOf?: Array<ReferenceOutput>;
+    period?: PeriodOutput;
+    phase?: CodeableConceptOutput;
+    primaryPurposeType?: CodeableConceptOutput;
+    principalInvestigator?: ReferenceOutput;
+    protocol?: Array<ReferenceOutput>;
+    reasonStopped?: CodeableConceptOutput;
+    resourceType: "ResearchStudy";
+    site?: Array<ReferenceOutput>;
+    sponsor?: ReferenceOutput;
+    status: string;
+    title?: string;
+}
+
+// @public
+export interface Resource extends Record<string, unknown> {
+    id?: string;
+    implicitRules?: string;
+    language?: string;
+    meta?: Meta;
+    resourceType: string;
+}
+
+// @public
+export interface ResourceOutput extends Record<string, any> {
+    id?: string;
+    implicitRules?: string;
+    language?: string;
+    meta?: MetaOutput;
+    resourceType: string;
+}
+
+// @public (undocumented)
+export interface Routes {
+    (path: "/radiology-insights/jobs/{id}", id: string): GetJob;
+    (path: "/radiology-insights/jobs"): CreateJob;
+}
+
+// @public
+export interface SampledData extends Element {
+    data?: string;
+    dimensions: number;
+    factor?: number;
+    lowerLimit?: number;
+    origin: Quantity;
+    period: number;
+    upperLimit?: number;
+}
+
+// @public
+export interface SampledDataOutput extends ElementOutput {
+    data?: string;
+    dimensions: number;
+    factor?: number;
+    lowerLimit?: number;
+    origin: QuantityOutput;
+    period: number;
+    upperLimit?: number;
+}
+
+// @public
+export interface SexMismatchInferenceOutput extends RadiologyInsightsInferenceOutputParent {
+    kind: "sexMismatch";
+    sexIndication: CodeableConceptOutput;
+}
+
+// @public
+export interface TimePeriod {
+    end?: Date | string;
```

Comment:
Could we start with just `Date` or `string` ? It's easy (read: non-breaking) to expand input types later

Reply:
We designed this to accommodate both `Date` and `string` types because, while the value is transmitted as a `string` over the wire, the additional metadata from the service specification (like Typespec or OpenAPI) allows us to also support the `Date` type directly. This approach offers convenience to users by leveraging the `Date` object's `toString` method, eliminating the need for manual conversion or mapping at runtime. Including both types ensures our interface remains true to the wire format while providing flexibility and ease of use.